README.md

```diff
@@ -16,12 +16,52 @@ A flexible JSON object linter with out of the box support for OpenAPI Specificat
 
 ## Installation
 
-```shell
+### API
+
+```bash
 npm install @stoplight/spectral
 ```
 
+### CLI
+
+```bash
+npm install -g @stoplight/spectral
+```
+
 Supports Node v8.3+ and modern browsers.
 
+### Executable binaries
+
+For users without Node and/or NPM/Yarn, we provide standalone packages for all major platforms:
+
+- x64 Windows
+- x64 MacOS
+- x64 Linux
+
+Note, the binaries are **not** auto-updatable, therefore you will need to download a new version on your own.
+
+Use of binaries is fairly easy.
+Once downloaded, you can proceed with the standard procedure for running any CLI tool.
+
+```bash
+./spectral-linux lint petstore.yaml
+```
+
+#### Installing binaries system-wide
+
+##### Linux
+
+As root, run:
+
+```bash
+mv ./spectral-linux /usr/local/bin/spectral
+```
+
+You may need to restart your terminal.
+Now, `spectral` command will be accessible in your terminal!
+
+Head over to releases for the latest binaries.
```

Comment:
Link it! :) 

aaaaand can we show OS X? Somebody can add Windows sometime if they want.

Reply:
Wish I could, but Spectral doesn't seem to have releases enabled in GitHub :(